### ISSUE TITLE
Typo?

### ISSUE THREAD

**Issue by Khev** [OctThe16th/PPO-Keras#3]
Hi there, thanks for sharing your code. I think there's an error on line 280 in main.py

_critic_loss = self.critic.fit([obs], [reward], batch_size=BATCH_SIZE, shuffle=True, epochs=EPOCHS, verbose=False)_

Shoudn't the critic be fitting to the discounted_returns  instead of the rewards? That is the line should read

_critic_loss = self.critic.fit([obs], [**discounted_returns**], batch_size=BATCH_SIZE, shuffle=True, epochs=EPOCHS, verbose=False)_

**Comment by LuEE-C** [comment]
On line 204 we call the function self.transform_reward() which transforms the content of the reward array into the discounted reward, hope that clarifies

**Comment by Khev** [comment]
Ah ya, that makes sense. Thanks!

**Comment by Khev** [comment]
Also, I noticed you didn't use target networks for the critic. Did you observe any instability in the learning as a result? Just curious!

**Comment by Linzecong** [comment]
> On line 204 we call the function self.transform_reward() which transforms the content of the reward array into the discounted reward, hope that clarifies

@LuEE-C @Khev 

I think in the _transform_reward_ function, at the end of the code, we should reverse the reward. 

Because there should be more rewards in the future, not in the past.

Your code is easy to understand. I'm looking forward to your reply.
